libyul/backends/evm/EVMDialect.cpp

```diff
@@ -152,7 +158,7 @@ map<YulString, BuiltinFunctionForEVM> createBuiltins(langutil::EVMVersion _evmVe
 				_assembly.appendDataOffset(_context.subIDs.at(dataName));
 			}
 		}));
-		builtins.emplace(createFunction("datacopy", 3, 0, false, false, false, false, [](
+		builtins.emplace(createFunction("datacopy", 3, 0, false, false, false, false, true, false, [](
```

Comment:
It should be fine. The last three flags are:
 - invalidates storage
 - invalidates memory
 - requires literal arguments

It is called datacopy instead of codecopy because we wanted to abstract away the fact that data is stored in code, since it might be different on other platforms.